### PR TITLE
Add default values to @Value annotations where possible

### PR DESCRIPTION
**Pull request by dsyer**
Without these you can't use zipkin-java-server as a library without
special knowledge of the YAML configuration. With them it is possible
to start a server without any configuration at all.

### CODE REVIEW

File: zipkin-java-server/src/main/java/io/zipkin/server/ZipkinQueryApiV1.java [hunk 1]
```diff
@@ -47,7 +49,7 @@
   private static final String APPLICATION_THRIFT = "application/x-thrift";
 
   @Autowired
-  @Value("${zipkin.query.lookback}")
+  @Value("${zipkin.query.lookback:86400000}")
   int defaultLookback = 86400000; // 7 days in millis
```
Comment (marcingrzejszczak) [inline zipkin-java-server/src/main/java/io/zipkin/server/ZipkinQueryApiV1.java]: I guess this value is not needed any more if you provide it via `@Value` ?

Comment (dsyer) [inline zipkin-java-server/src/main/java/io/zipkin/server/ZipkinQueryApiV1.java]: Technically true, but only in a Spring context. There might be unit tests that need a value (there aren't, but I was leaving it initialized to be defensive).

Comment (marcingrzejszczak) [inline zipkin-java-server/src/main/java/io/zipkin/server/ZipkinQueryApiV1.java]: +1

File: zipkin-java-server/src/main/java/io/zipkin/server/ZipkinQueryApiV1.java [hunk 2]
```diff
@@ -111,7 +113,7 @@ public void uploadSpansThrift(@RequestBody byte[] body) {
         .minDuration(minDuration)
         .maxDuration(maxDuration)
         .endTs(endTs)
-        .lookback(lookback != null ? lookback : defaultLookback)
+        .lookback(lookback != null ? lookback : this.defaultLookback)
```
Comment (adriancole) [inline zipkin-java-server/src/main/java/io/zipkin/server/ZipkinQueryApiV1.java]: argh.. this. is back

**Comment by adriancole** [comment]
I'm going to pick this in, scrubbing the `this.` and import changes. Thanks for the info on how to get this to work!

**Comment by adriancole** [comment]
Thanks for fixing this. releasing now as 0.4.1

**Pull request closed without merging**